rust/libnewsboat/src/utils.rs

```diff
@@ -36,6 +36,15 @@ pub fn to_u(rs_str: String, default_value: u32) -> u32 {
     result.unwrap()
 }
 
+pub fn trim(rs_str: String) -> String {
+    rs_str.trim().to_string()
+}
+
+pub fn trim_end(rs_str: String) -> String {
+    let x: &[_] = &['\n','\r'];
+    rs_str.trim_right_matches(x).to_string()
```

Comment:
[According to the docs](https://doc.rust-lang.org/std/primitive.str.html#method.trim_right_matches), `trim_right_matches` is deprecated in favor of `trim_end_matches` starting from version 1.33.0. I'm not sure, which Rust version we require at the moment.

Reply:
I haven't decided on the required version yet, but I'm pretty sure it'll be something ancient like 1.23 (released in January 2018), because it doesn't seem like distros take kindly to Rust's six-week release schedule.

Reply:
Which function should I use? The reason I used `trim_right_matches()` is my distro did not have a version of rustc updated enoughto have `trim_end_matches()` until this morning. So now I'm really not bothered.

Reply:
Keep `trim_right_matches` for now. We'll replace it with `trim_end_matches` when we update the project to newer Rust versions.